pkg/watch/health.go

```diff
@@ -186,7 +200,7 @@ func (p *PodWatch) updateNeeded(res health.Result, ttl time.Duration) bool {
 
 // compares services being monitored with services that
 // need to be monitored.
-func updatePods(current []PodWatch, reality []kp.ManifestResult, logger *logging.Logger, store kp.Store, node string) []PodWatch {
+func updatePods(store kp.Store, client *http.Client, current []PodWatch, reality []kp.ManifestResult, node string, logger *logging.Logger) []PodWatch {
```

Comment:
nit: Now that there are so many arguments here, can you switch formatting styles? This will help keep future diffs small and precise, and there is less room for error putting new arguments in the wrong slot (like I just did in my other diff).

```
func updatePods(
    store kp.Store,
    ...,
    logger *logging.Logger,
) []PodWatch {
```